Cargo.toml

```diff
@@ -21,6 +21,7 @@ backon = { version = "1.5.1", features = ["tokio-sleep"] }
 borsh = { version = "1.5.1", features = ["derive"] }
 clap = { version = "4.5.20", features = ["derive", "env"] }
 curve25519-dalek = "4.1.3"
+# TODO(#851): Use upstream library
```

Comment:
I can't find the issue #851.

<img width="736" height="433" alt="screenshot" src="https://github.com/user-attachments/assets/1b519f60-509d-4aab-8302-672169ca83b1" />